pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -290,6 +295,61 @@ func selectVirtualServices(virtualServices []config.Config, servicesByName map[h
 	return out
 }
 
+type ProxyHeaders struct {
+	ServerName                 string
+	ServerHeaderTransformation hcm.HttpConnectionManager_ServerHeaderTransformation
+	ForwardedClientCert        hcm.HttpConnectionManager_ForwardClientCertDetails
+	IncludeRequestAttemptCount bool
+	GenerateRequestID          *wrappers.BoolValue
```

Comment:
nit: pointer to pointer with bool value is weird, either *bool, or value would be better.

Reply:
The intent is to have this be pre-processed so we can just stick it into the protobuf. Proto-go requires pointers unfortunately. Agree its weird